portal-impl/src/com/liferay/portal/util/PortalImpl.java

```diff
@@ -2600,42 +2449,6 @@ else if (type == ExpandoColumnConstants.STRING_LOCALIZED) {
 		return value;
 	}
 
-	@Override
-	public Serializable getExpandoValue(
```

Comment:
@slnn It seems you are removing a lot of methods, are they related to Expando, or you just want a complete clean-up?

Reply:
Actually, Expando just needs to remove part of them, but I want a complete clean-up.

Reply:
I have updated and retriggered ci tests